### ISSUE TITLE
[4.1] Dropwdown menu to open backend or frontend in a new window

### ISSUE THREAD

**Issue by sandewt** [joomla/joomla-cms#31460]
### New feature [4.1]

This is an improvement proposal for [4.1].

### Expected result

Open in a dropwdown menu the backend or frontend in a new window. See image
![screen shot 2020-11-22 at 14 31 53](https://issues.joomla.org/uploads/1/8a258e61a3ffd92b1ffa2fced386288f.png)

### Actual result

It's in [4.0] only possible to open the frontend in a new window.

### System information (as much as possible)

Irrelevant

### Additional comments

In [3.x] is it optional to open the backend in a new window.

**Comment by C-Lodder** [comment]
You know that opening another instance of the backend can be done by **clicking** the middle mouse button on the logo, right?
And `CTRL + Enter` for keyboard users
Just saying...

**Comment by brianteeman** [comment]
I dont understand your additional coments

**Comment by sandewt** [comment]
>I dont understand your additional coments

Screenshots J3.9.x

![vieuws-2](https://user-images.githubusercontent.com/9271775/99946015-f46ade00-2d75-11eb-94bc-c0ba0fb0e016.png)
![vieuws-3](https://user-images.githubusercontent.com/9271775/99946024-f6cd3800-2d75-11eb-8da2-4e6c5da88e69.png)

**Comment by ReLater** [comment]
@sandewt 

If your psoposal is not accepted you can use https://github.com/GHSVS-de/mod_administratorlinkghsvs#readme

I use the version with position "status".

![23-11-_2020_14-01-17](https://user-images.githubusercontent.com/20780646/99965030-8e418380-2d94-11eb-8521-21dc88ab097c.jpg)

**Comment by bembelimen** [comment]
The Module from @ReLater looks really good, perhaps we can convince him to put it into the JED (if not done already).

Do you think, there is something left which would benefit, when we put it into the core or does the module from @ReLater covers everything?

**Comment by chmst** [comment]
My2cent: I would prefer calling the frontend with one click as it is now.

**Comment by brianteeman** [comment]
I really dont see the need for this. Being able to open the frontend with one click is a useful addition. I can already open the admin with one click - anyone responsible for building a web site should be expected to know how to use their browser

**Comment by ReLater** [comment]
> I can already open the admin with one click

You can't in an edit view. And that was possible in J3 via User Status module which opens a NEW tabulator. The administration link (above, left) is deactivated in Joomla 4 in edit views. Therefore you can't click it at all there and CAN not open a new tab with holding STRG. You have to close the edit view, open a new tab, reopen the edit view.... Or you can clone the edit view tabulator, close the new edit and check the other edit view out in the database in Joomla 4.

**Comment by sandewt** [comment]
![vieuws-4](https://user-images.githubusercontent.com/9271775/100088007-d376bb80-2e4f-11eb-841b-1f429db91468.png)

I confirm that it is here (see screenshot) not possible to open a new browser tab with _Ctrl + one mousèclick_. 

I think it is useful, according to the situation in J3, to change this through a separate PR.

**Comment by sandewt** [comment]
>My2cent: I would prefer calling the frontend with one click as it is now.

This means a separate solution for the admin view and not a dropdwon menu, similar (= new module) to that of @ReLater.

**Comment by ReLater** [comment]
> The Module from @ReLater looks really good, perhaps we can convince him to put it into the JED (if not done already).

Published now in JED. https://extensions.joomla.org/extension/administration/admin-navigation/administration-in-new-tab-by-ghsvs-de/

**Comment by sandewt** [comment]
>The Module from @ReLater looks really good, perhaps we can convince him to put it into the JED (if not done already).

Looks good in JED, unfortunately it is not included in Joomla 4 by default.

@relater can / do you want to make a PR?

**Comment by ReLater** [comment]
> can / do you want to make a PR?

I was asked here to publish it in JED. Now it shall go into core? Would be a waste of time for me. And if I want to add a new feature I have to wait until Joomla version xyz ;-)

**Comment by sandewt** [comment]
> I was asked here to publish it in JED. Now it shall go into core? Would be a waste of time for me. And if I want to add a new feature I have to wait until Joomla version xyz ;-)


I know from other users that there is a need for this feature in Joomla. Unfortunately, they have not reported here.

**Comment by ReLater** [comment]
The JED is public. The extension is free, GPL and OpenSource. The download link is a direct one. It can be installed via web installer. It has an issue tracker and so on. There are many extensions in JED where it would be nice to have them in core; for some users.

And to be honest: In decades of Joomla administrations for friends and customers I was always the only one who activated and used that feature in these Joomla 3 back ends or, who installed the previous version for older Joomla versions  ;-) ;-) ;-)

In other words: I personally see no need to have this module in the core. Would be just one more core extension that will be used seldomly. You are also forgetting the rat's tail that core extensions make necessary, e.g. with regard to database adjustments.

And it can't go into Joomla 4.0, if I understand the guidelines regarding new features correctly.

**Comment by brianteeman** [comment]
Is there any point in this staying open? @bembelimen
